drivers/src/memory_layout.rs

```diff
@@ -63,13 +68,15 @@ pub const RTALIAS_TBS_SIZE: u32 = 1024;
 pub const PCR_LOG_SIZE: u32 = 1024;
 pub const MEASUREMENT_LOG_SIZE: u32 = 1024;
 pub const FUSE_LOG_SIZE: u32 = 1024;
-pub const DPE_SIZE: u32 = 4 * 1024;
+pub const DPE_SIZE: u32 = 5 * 1024;
 pub const PCR_RESET_COUNTER_SIZE: u32 = 1024;
 pub const DATA_SIZE: u32 = 78 * 1024;
 pub const STACK_SIZE: u32 = 22 * 1024;
 pub const ROM_STACK_SIZE: u32 = 14 * 1024;
-pub const ESTACK_SIZE: u32 = 1024;
```

Comment:
I think I would prefer to shrink the data size rather than change the stack size. I don't think we're using nearly that much data.

Reply:
I agree...reducing stack size is risky...